src/test/java/org/broadinstitute/consent/http/resources/MetricsResourceTest.java

```diff
@@ -30,7 +30,8 @@ public void setUp() {
     MockitoAnnotations.initMocks(this);
   }
 
-  private void initResource() {
+  @Before
```

Comment:
We actually don't want this to run before every test invocation. It is intended to be called only by tests after they have done all of the initialization they need. The net result of adding a `@Before` here is that we initialize the resource twice for every test instead of only once.